packages/measured-reporting/README.md

```diff
@@ -15,20 +15,96 @@ npm install measured-reporting
 ### [Self Reporting Metrics Registry](https://yaorg.github.io/node-measured/SelfReportingMetricsRegistry.html)
 A dimensional aware self-reporting metrics registry, just supply this class with a reporter implementation at instantiation and this is all you need to instrument application level metrics in your app.
 
+See the [SelfReportingMetricsRegistryOptions](http://localhost:63342/node-measured/build/docs/packages/measured-reporting/global.html#SelfReportingMetricsRegistryOptions) for advanced configuration.
+
+```javascript
+const { SelfReportingMetricsRegistry, LoggingReporter } = require('measured-reporting');
+const registry = new SelfReportingMetricsRegistry(new LoggingReporter({
+  defaultDimensions: {
+    hostname: os.hostname()
+  }
+}));
+
+// The metric will flow through LoggingReporter#_reportMetrics(metrics) every 10 seconds by default
+const myCounter = registry.getOrCreateCounter('my-counter');
+
+```
+
 ### [Reporter Abstract Class](https://yaorg.github.io/node-measured/Reporter.html)
-The base class for reporter implementations. This class is extended and the [_reportMetrics(metrics)](https://yaorg.github.io/node-measured/Reporter.html#_reportMetrics__anchor) method gets overridden to create an data aggregator specific reporter. See the [SignalFx Reporter](../measured-signalfx-reporter/) for an example implementation.
+The base class for reporter implementations. This class is extended and the [_reportMetrics(metrics)](https://yaorg.github.io/node-measured/Reporter.html#_reportMetrics__anchor) method gets overridden to create an data aggregator specific reporter. 
 
+See the [ReporterOptions](http://localhost:63342/node-measured/build/docs/packages/measured-reporting/global.html#ReporterOptions) for advanced configuration.
+
+#### Current Implementations
+- [SignalFx Reporter](https://yaorg.github.io/node-measured/SignalFxMetricsReporter.html) in the `measured-signalfx-reporter` package.
+  - reports metrics to SignalFx.
+- [Logging Reporter](https://yaorg.github.io/node-measured/LoggingReporter.html) in the `measured-reporting` package.
+  - logs reported metrics at an info level through a supplied logger, defaulting to a new bunyan logger if none supplied.
```

Comment:
Maybe:

logs ~reported~ metrics at an info level through a supplied logger, defaulting to a new bunyan logger if none supplied.